README.md

```diff
@@ -193,14 +182,11 @@ for a more in-depth example, showcasing a Single Page Application mode.
   <head>
     <meta charset="UTF-8">
     <title>Sample FirebaseUI App</title>
-    <!-- *******************************************************************************************
-       * TODO(DEVELOPER): Paste the initialization snippet from:
-       * Firebase Console > Overview > Add Firebase to your web app.
-       * In addition, include the firebase-auth SDK:
-       * <script src="https://www.gstatic.com/firebasejs/[FIREBASE VERSION USED IN SNIPPET]/firebase-auth.js"></script> *
-       ***************************************************************************************** -->
-    <script src="https://www.gstatic.com/firebasejs/ui/5.0.0/firebase-ui-auth.js"></script>
-    <link type="text/css" rel="stylesheet" href="https://www.gstatic.com/firebasejs/ui/5.0.0/firebase-ui-auth.css" />
+    <script src="https://www.gstatic.com/firebasejs/9.1.2/firebase-app-compat.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/9.1.2/firebase-auth-compat.js"></script>
+    <script src="/__/firebase/init.js"></script>
```

Comment:
The version will keep updating (it is 9.1.3 now). Instead let's keep the old instruction on how to "Add Firebase to your web app".
Also not all customers use Firebase Hosting (for example, many GCIP customers do not use it). You can add a note on `<script src="/__/firebase/init.js"></script>` for hosting customers.